src/codeu/chat/server/Controller.java

```diff
@@ -60,8 +69,68 @@ public Message newMessage(Uuid id, Uuid author, Uuid conversation, String body,
     final Conversation foundConversation = model.conversationById().first(conversation);
 
     Message message = null;
+    Connection connection = null;
```

Comment:
It feels like all this logic should be in the model. In the interest of getting this particular review done I think it is fine to have it here, but I think in a followup change you should move the database code like this into the model. I'm not really seeing the benefit you are getting from putting it here.